src/app/cube/user-profile/user-preferences/user-preferences.component.html

```diff
@@ -1,5 +1,5 @@
-<!-- <app-contextmenu [content]="modalCtrl.contextMenuContent"></app-contextmenu>
-<app-dialogmenu [content]="modalCtrl.dialogMenuContent"></app-dialogmenu> -->
+<!-- <clark-contextmenu [content]="modalCtrl.contextMenuContent"></clark-contextmenu>
+<clark-dialogmenu [content]="modalCtrl.dialogMenuContent"></clark-dialogmenu> -->
```

Comment:
If this is commented out then shouldn't it be deleted?

Reply:
Good catch! I didn't notice this one because I did a bulk find and replace

Reply:
This component looked a little odd to me, so I looked into it a little more and it seems that it was never fully built out:
![image](https://user-images.githubusercontent.com/14933100/98695057-0fc1fc00-2327-11eb-9e05-72b2c318041d.png)

@pzalep1 it might be a good idea for this to be removed in a future ticket